uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/test/manager-leave/components/manager-leave-calendar_test.js

```diff
@@ -427,6 +423,12 @@
           return data.type.name === dayType;
         });
       }
+
+      function filterOptionGroup(optionGroupName, property, value) {
```

Comment:
can you move this method to `optionGroupMock` so that it becomes available to other files too?